apps/server/src/app/export/msword.ts

```diff
@@ -40,6 +58,225 @@ export class MSWordExporter {
+            this.logger.log(" STRUCTURE title :" + structureDefinition.title);
+
+
+            this.logger.log(" STRUCTURE version:" + structureDefinition.version);
+
+            this.logger.log(" STRUCTURE fhirversion:" + structureDefinition.fhirVersion);
+
+          this.logger.log(" STRUCTURE publisher:" + structureDefinition.publisher);
+
+          this.logger.log(" STRUCTURE contact:" + structureDefinition.contact);
+
+          this.logger.log(" STRUCTURE copyright:" + structureDefinition.copyright);
+
+
+          // const structureDefinition: StructureDefinition = <StructureDefinition> entry.resource;  // we can do this because we know entry.resource will be a StructureDefinition because of the filter
+
+          this.body.push(new Paragraph({ text: `Name: ${structureDefinition.name}` }));    // name is required, so we can expect it to be populated with a value
+          if (structureDefinition.title) {
+            this.body.push(new Paragraph({ text: `Title: ${structureDefinition.title}` }));    // name is required, so we can expect it to be populated with a value
+          }
+
+
+          if (structureDefinition.differential) {
+            (structureDefinition.differential.element || []).forEach((element, eIndex) => {
+                if(element.id === 'Extension'){
+                  this.body.push(new Paragraph({ text: `Element ${eIndex + 1}` }));
+                  this.body.push(new Paragraph({ text: `Id ${element.id}` }));
+                  this.body.push(new Paragraph({ text: `Path: ${element.path}` }));
+                }
+                if(element.id === 'Extension.url') {
+                  this.body.push(new Paragraph({ text: `Element ${eIndex + 1}` }));
+                  this.body.push(new Paragraph({ text: `Extension.url ${element.id}` }));
+                  this.body.push(new Paragraph({ text: `Path: ${element.path}` }));
+                }
+                if(element.id === 'Extension.value[x]') {
+                  this.body.push(new Paragraph({ text: `Element ${eIndex + 1}` }));
+                  this.body.push(new Paragraph({ text: `Extension.value[x] ${element.id}` }));
+                  this.body.push(new Paragraph({ text: `Path: ${element.path}` }));
+                }
+                this.body.push(new Paragraph({ text: `Fixed: ${element.fixed}` }));
+                this.body.push(new Paragraph({ text: `Min: ${element.min}` }));
+                this.body.push(new Paragraph({ text: `Max: ${element.max}` }));
+                this.body.push(new Paragraph({ text: `Must support: ${element.mustSupport}` }));
+                this.body.push(new Paragraph({ text: `Binding: ${element.binding}` }));
+                this.body.push(new Paragraph({ text: `Short: ${element.short}` }));
+                this.body.push(new Paragraph({ text: `Label: ${element.label}` }));
+
+              }
+            );
+          };
+
+
+
+            keys.forEach((key) => {
```

Comment:
I like the way the information is output above, showing each important property, rather than just dumping out all the keys. I would suggest removing this. Also, I suggest creating a table for each element like you do here and putting "fixed", "min", "max", "mustSupport", etc. in the table as a individual rows. I also suggest checking the existence of those properties first, so that we don't output empty rows. Tables are a nice way of displaying repeating data, and can be applied to code below this as well for ValueSet and CodeSystem.